### ISSUE TITLE
Track Users cursors in Shared Workspaces

### ISSUE THREAD

**Issue by ChevronTango** [gitpod-io/gitpod#18947]
## Is your feature request related to a problem? Please describe

It can be quite hard to follow users  when sharing workspaces. They may be making changes in many different files, or the same file as you, and it can be hard to follow them.

## Describe the behaviour you'd like

A cursor, similar to that used in LiveShare, which shows where other users are looking and editing at any given moment. It would also be convenient to have the ability to "follow" a user, so that you will always be looking at the file and line they are editing, to aid with pair programming.

**Comment by loujaybee** [comment]
x-posting related comment: 

> Thanks for the feedback, @ChevronTango - more granular control on workspace sharing would be great.
It's worth noting that workspace sharing (if required) can be disabled at the organization level:
https://www.gitpod.io/changelog/organizational-policy-workspace-sharing
And many 3rd party sharing plugins also work (if that's use to anyone looking for more feature-rich solutions):
https://www.gitpod.io/docs/configure/workspaces/collaboration#external-collaboration-plugins

https://github.com/gitpod-io/gitpod/issues/18945#issuecomment-1769043176
